db/database_test.go

```diff
@@ -1153,7 +1147,7 @@ func TestRecentSequenceHistory(t *testing.T) {
 		revid, err = db.Put("doc1", body)
 		// Sleep needed to ensure consistent results when running single-threaded vs. multi-threaded test:
 		// without it we can't predict the relative update times of nextSequence and RecentSequences
-		time.Sleep(5 * time.Millisecond)
+		time.Sleep(500 * time.Millisecond)
```

Comment:
Can we bump to 50 instead - otherwise this is adding 10s to total test time.

If it starts failing, leave at 500 and file a followup ticket to clean up this test in general.